app/models/form.rb

```diff
@@ -142,4 +144,11 @@ def is_last_question?(page, subsection, case_log)
     subsection_ids = subsections.map(&:id)
     subsection.id == subsection_ids[subsection_ids.length - 1] && next_page(page, case_log) == :check_answers
   end
+
+  def previous_page(page_ids, page_index, case_log)
```

Comment:
Does this work for all cases? Could we use it for back button functionality in the form pages? (Rather than the current js history call)

Worth a spec as well I think?

Reply:
we only call it after the interruption screen but will see if it would make sense as the back button

Reply:
After looking at it I'd be keen to leave this out for now, because there's a lot of code duplication that would need to be refactored, how important do we think it is?

Reply:
Important (for accessibility) but definitely doesn't need to be in this PR.

Reply:
Would still be good to have some tests for this method though